argocd_applications/creating-an-application-using-gitops-argocd-cli.adoc

```diff
@@ -0,0 +1,22 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="creating-an-application-using-gitops-argocd-cli"]
+= Creating an application by using the GitOps CLI
+:context: creating-an-application-using-gitops-argocd-cli
+
+toc::[]
+
+With Argo CD, you can create your applications to the OpenShift cluster either by using the {gitops-shortname} `argocd` CLI.
```

Comment:
This statement looks confusing and incomplete.
Can we just have it as `you can create applications on the OpenShift cluster using the {gitops-shortname} `argocd` CLI.`

Reply:
@anandf Good catch, I have addressed it.